test/phoenix_live_view/html_tokenizer_test.exs

```diff
@@ -64,11 +65,75 @@ defmodule Phoenix.LiveView.HTMLTokenizerTest do
 
       assert [
                {:tag_open, "p", [], %{line: 1, column: 1}},
-               {:text, "\n<!--\n<div>\n-->\n", %{line_end: 5, column_end: 1}},
+               {:text, "\n<!--\n<div>\n-->\n",
+                %{line_end: 5, column_end: 1, context: [:comment_start, :comment_end]}},
```

Comment:
@josevalim my code is not yet 100% I just wanted to update the test cases to make sure what we are expecting here. 

If I get it right, we want to add `comment_start` and `comment_end` when it is inline comments. Does it make sense? 🤔

Reply:
In this case they should cancel out IMO and there is no context. Unless you need to know that there was a comment at some point?

Reply:
This would be nice to have but as just `:comment`. But I wouldn't worry checking it in the formatter since that would be a simple check